scripts/configs/data/data_params.yaml

```diff
@@ -9,7 +9,8 @@ image_resize_dims:
   height: 256
 
 # ABSOLUTE path to data directory. the below is just a toy example for running Getting Started scripts
-data_dir: "toy_datasets/toymouseRunningData"
+# data_dir: "/home/jovyan/datastores/mirror-mouse"
+data_dir: "/datastores/mirror-mouse"
```

Comment:
revert to original please `data_dir: "toy_datasets/toymouseRunningData"` -- this is needed for the demos